src/hevm/README.md

```diff
@@ -58,20 +58,37 @@ set the `TERMINFO` environment variable; on Ubuntu, you should do
 
 ### Building with Stack or Cabal
 
-If you can't or won't use Nix, the easiest way especially if you don't
-have GHC (the Haskell compiler) installed already, is to use
-[Stack](https://docs.haskellstack.org/en/stable/README/), which can
-take care of installing GHC for you.  These commands should work:
-
-    $ curl -sSL https://get.haskellstack.org/ | sh
-    $ git clone https://github.com/dapphub/hevm
-    $ cd hevm && stack setup && stack install
-
-If you prefer to use your own installation of GHC and the basic
-Haskell package manager, Cabal, simply run:
-
-    $ git clone https://github.com/dapphub/hevm
-    $ cd hevm && cabal configure && cabal install
+If you can't or won't use Nix, the easiest way especially if you don't have GHC (the Haskell compiler) installed already, is to use [Stack](https://docs.haskellstack.org/en/stable/README/), which can take care of installing GHC for you.  These commands should work:
```

Comment:
Why remove the hard linebreaks? I quite like them in markdown files, as they make navigation with vim much easier.

Reply:
Oh, there isn't a reason. I can break them and request a new pull request.

Reply:
Could you just undo these changes and push to this branch? So we can stay on the same PR.